hudi-integ-test/src/test/java/org/apache/hudi/integ/ITTestHoodieDemo.java

```diff
@@ -288,12 +290,15 @@ private void testHiveAfterFirstBatch() throws Exception {
 
   private void testSparkSQLAfterFirstBatch() throws Exception {
     Pair<String, String> stdOutErrPair = executeSparkSQLCommand(SPARKSQL_BATCH1_COMMANDS, true);
-    assertStdOutContains(stdOutErrPair, "|default |stock_ticks_cow   |false      |\n"
-                                                    + "|default |stock_ticks_cow_bs   |false      |\n"
-                                                    + "|default |stock_ticks_mor_bs_ro |false      |\n"
-                                                    +  "|default |stock_ticks_mor_bs_rt |false      |"
-                                                    + "|default |stock_ticks_mor_ro |false      |\n"
-                                                    +  "|default |stock_ticks_mor_rt |false      |");
+    assertStdOutContains(stdOutErrPair,
+        "|default  |stock_ticks_cow      |false      |\n"
+            + "|default  |stock_ticks_cow_bs   |false      |\n"
+            + "|default  |stock_ticks_mor      |false      |\n"
```

Comment:
Now the original table name is also synced to the metastore based on the recent behavior change (#10685).